src/sentry/api/endpoints/relay/project_configs.py

```diff
@@ -64,6 +64,7 @@ def _post(self, request: Request):
 
         if version == "3" and request.relay_request_data.get("global"):
             response["global"] = get_global_config()
+            response["global_status"] = "ready"
```

Comment:
I believe `global_status` goes inside `global` config, so we should move this to `get_global_config` (the failing test complains about this I think).

Reply:
This is a new field added in https://github.com/getsentry/relay/pull/2765 which goes on the same level